benchmarks/initial.bench.ts

```diff
@@ -0,0 +1,20 @@
+import { bench, describe } from 'vitest';
+import { initial as lodashInitial } from 'lodash';
+import { initial as esToolkitInitial } from 'es-toolkit'; // hypothetical library
+
+// Helper function to generate a large array
+function generateLargeArray(size) {
+  return Array.from({ length: size }, (_, index) => index);
+}
+
+describe('initial function performance', () => {
+  const largeArray = generateLargeArray(1000000); // 1 million elements
+
+  bench('es-toolkit/initial', () => {
+    esToolkitInitial(largeArray);
+  });
+
+  bench('lodash/initial', () => {
+    lodashInitial(largeArray);
+  });
+});
```

Comment:
```suggestion
import { bench, describe } from 'vitest';
import { initial as initialLodash } from 'lodash';
import { initial as initialToolkit } from 'es-toolkit'; // hypothetical library

// Helper function to generate a large array
function generateLargeArray(size) {
  return Array.from({ length: size }, (_, index) => index);
}

describe('initial function performance', () => {
  const largeArray = generateLargeArray(1000000); // 1 million elements

  bench('es-toolkit/initial', () => {
    initialToolkit(largeArray);
  });

  bench('lodash/initial', () => {
    initialLodash(largeArray);
  });
});
```

I think it's better for us to follow other benchmarks.